password_security/controllers/__init__.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=missing-docstring
```

Comment:
This shouldn't be needed. Remove this line. If it fails, then it should be fixed in MQT.